packages/endpoint/src/endpoint.d.ts

```diff
@@ -143,7 +143,7 @@ export interface EndpointInstance<
     : IfAny<M, any, IfTypeScriptLooseNull<'read', 'mutate'>>;
 
   /** @deprecated */
-  getFetchKey(params: Parameters<F>[0]): string;
+  getFetchKey(params: Parameters<F>): string;
```

Comment:
is this the right type for this? it kind of seems like you want this to take varargs, but this means instead of taking a single thing, it now takes an array of things?

Reply:
good call

Reply:
added the same test that key() had to getFetchKey()